messages.md

```diff
@@ -213,6 +277,10 @@ FIXME: see https://github.com/re-vault/practical-revault/issues/15
                 (....)
     ||   -- get_spend_opinions -->   ||
     ||  <--- spend_opinions  -----   ||  // Eventually all watchtowers responded.
+              (..sig..)
+    ||   -- finalize_spend  ----->   ||  // Ok we and the cosigners signed, here is the tx
+    ||   - get_spend_validations ->  ||  // Do watchtowers let me be now ?
+    ||   -- spend_validations --->   ||  // Yep, go ahead.
```

Comment:
spend_validations goes from SS to manager

Reply:
Think i did not have coffee when i wrote this.